src/AddIns/Uno.UI.Svg/Uno.UI.Svg.netcoremobile.csproj

```diff
@@ -65,6 +65,15 @@
 
 	<Import Project="..\..\SourceGenerators\Uno.UI.SourceGenerators\Content\Uno.UI.SourceGenerators.props" />
 
+	<Target Name="_RemoveUnoPackageAnalyzers"
+			BeforeTargets="CoreCompile;XamlPreCompile">
+		<!-- Remove analyzer references that are included in the PackageReference above -->
+		<ItemGroup>
+			<_AnalyzerToRemove Include="@(Analyzer)" Condition="'%(Analyzer.NuGetPackageId)'=='Uno.UI' or '%(Analyzer.NuGetPackageId)'=='Uno.WinUI' " />
+			<Analyzer Remove="@(_AnalyzerToRemove)"/>
+		</ItemGroup>
+	</Target>
```

Comment:
Why do we need this? If we are getting duplicate analyzer references, I think we should instead remove line 66:

```xml
	<Import Project="..\..\SourceGenerators\Uno.UI.SourceGenerators\Content\Uno.UI.SourceGenerators.props" />
```

Reply:
We need to be running the local version of the generators, not the ones that are coming from the referenced nuget package. For some reason, the `ResolvePackageAssets` is still returning Analyzers even if everything's disabled:

https://github.com/unoplatform/uno/blob/7ff5cefdc3f38991b73797c197a54efbc8b60e6c/src/AddIns/Uno.UI.Svg/Uno.UI.Svg.netcoremobile.csproj#L41

This feels like a bug in recent version of .NET, but not too sure what is causing it. It's an internal change anyways, so we can revert once it gets fixed.

Reply:
Oh I see. So the expectation is that we don't need this in favor of `ExcludeAssets="all" IncludeAssets="none"` but that's not working as expected.

It's definitely worth filing a bug for that (I think to NuGet/Home repo).